src/compile.ts

```diff
@@ -1,15 +1,24 @@
-import ts, { Statement } from "typescript";
+import ts from "typescript";
 import { PluginConfig, TransformerExtras } from "ts-patch";
 import { BinaryOp } from "./expression";
 import { AnyTable } from "./table";
 import { AnyLambda } from "./function";
 import { FunctionlessNode } from "./node";
+import { AppsyncResolver } from "./appsync";
+import glob from "glob";
+import { assertDefined } from "./assert";
+
 export default compile;
 
 /**
  * Configuration options for the functionless TS transform.
  */
-export interface FunctionlessConfig extends PluginConfig {}
+export interface FunctionlessConfig extends PluginConfig {
+  /**
+   * Glob to exclude
+   */
+  exclude?: string;
```

Comment:
should this be an array?

Reply:
My original thought was that glob supports crazy things, so keep it singular, but looks like multiple is easy, so lets do that. And it'll match `tsconfig`.
https://www.typescriptlang.org/tsconfig#include